config/routes.rb

```diff
@@ -12,6 +12,8 @@
   get "/422", to: "errors#unprocessable_entity", via: :all
   get "/500", to: "errors#internal_server_error", via: :all
 
+  get "/trainees/not-supported-route", to: "trainees#not_supported_route"
```

Comment:
Does this route need to be nested within the trainees path? or can it be a global route on it's own?

Reply:
done 2a1ae83